### PR TITLE
Polished Akinator

### PR DESCRIPTION
**Pull request by NeginsCode**
- Exit the game using unexpected inputs like: "please let me finally exit this game" or "fraglabeldub"
- Redundant property was removed
- nlp:yes-no signal was removed and the property nlp:yesno gets set to "_" if input does not show "yes", "no", "i don't know", etc. 

Happy Hacking in the Hackquarters!

### CODE REVIEW

File: modules/ravestate_akinator/__init__.py [hunk 1]
```diff
@@ -148,21 +141,34 @@ def is_it_answered(ctx):
             out = "Yeah! I guessed right! Thanks for playing with me! \nDo you want to play again?"
```
Comment (Toseban) [inline modules/ravestate_akinator/__init__.py]: The \n might be a bad idea for speech synthesis

File: modules/ravestate_akinator/__init__.py [hunk 2]
```diff
@@ -148,21 +141,34 @@ def is_it_answered(ctx):
...
+    @state(cond=s("nlp:yesno:changed") & s("akinator:exit-game", max_age=13.),
+           write=("rawio:out", "akinator:question"),
+           read="nlp:yesno",
+           emit_detached=True)
+    def exit_game_answered(ctx):
+        if ctx["nlp:yesno"] == "yes":
+            ctx["rawio:out"] = "Thanks for playing with me!"
+            return Resign()
+        else:
+            ctx["rawio:out"] = "Yeah! Let's keep playing! Remember that you have these five answering choices:" \
```
Comment (Toseban) [inline modules/ravestate_akinator/__init__.py]: same as before, maybe adapt a little bit for better synthesis

File: modules/ravestate_akinator/__init__.py (hunk 1 above)
Comment (NeginsCode) [reply modules/ravestate_akinator/__init__.py]: True :) it's gone

File: modules/ravestate_akinator/__init__.py (hunk 2 above)
Comment (NeginsCode) [reply modules/ravestate_akinator/__init__.py]: :)

**Pull request merged**